script.js

```diff
@@ -99,6 +110,10 @@ restaurantsApp.init = function () {
 }
 
 
-$(function() {
+$(async function() {
     restaurantsApp.init()
+
+    // Get user city and populate the locationTerm input
+    let userCity = await restaurantsApp.getCity();
+    $('#locationInput').val(userCity)
```

Comment:
can you explain also here why the document ready has became an async function. Also the getcity function should be called in the init function. the only function called in document ready is the init.